.github/workflows/commit.yml

```diff
@@ -3,6 +3,9 @@ on:
   push:
     branches-ignore:
       - 'dependabot/*'
+  
+  pull_request:
+    branches: [ master ]
```

Comment:
Please split the commit check in another PR so it can go in fast; also this PR is changing two different things which are not a good practice and we should avoid whenever possible.

Reply:
Removed.